### PR TITLE
fix(clerk-js): Cleanup org profile forms

### PR DESCRIPTION
**Pull request by desiprisg**
## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

File: packages/clerk-js/src/ui/common/RemoveResourcePage.tsx [hunk 1]
```diff
@@ -39,14 +44,10 @@ export const RemoveResourcePage = withCardStateProvider((props: RemovePageProps)
...
             variant='primaryDanger'
-            onReset={close}
+            onReset={onReset || close}
           />
         </Form.Root>
       </FormContent>
-      <SuccessPage
-        title={title}
-        text={successMessage}
-      />
```
Comment (anagstef) [inline packages/clerk-js/src/ui/common/RemoveResourcePage.tsx:49]: Are we removing this?

Comment (desiprisg) [reply packages/clerk-js/src/ui/common/RemoveResourcePage.tsx:49]: No designs for it. I'll remove those from the codebase in another PR.

File: packages/clerk-js/src/ui/components/OrganizationProfile/ProfileForm.tsx [hunk 2]
```diff
@@ -79,41 +76,34 @@ export const ProfileForm = withCardStateProvider((props: ProfileFormProps) => {
...
+            {...slugField.props}
+            onChange={onChangeSlug}
           />
-        </Form.Root>
-      </FormContent>
-      <SuccessPage
-        title={title}
-        text={localizationKeys('organizationProfile.profilePage.successMessage')}
-        onFinish={onSuccess}
-      />
```
Comment (anagstef) [inline packages/clerk-js/src/ui/components/OrganizationProfile/ProfileForm.tsx:116]: Should we remove this localization key?

**Review by panteliselef — APPROVED** [review]

File: packages/clerk-js/src/ui/components/OrganizationProfile/RemoveDomainForm.tsx [hunk 3]
```diff
@@ -66,17 +64,14 @@ export const RemoveDomainForm = (props: RemoveDomainFormProps) => {
...
-        domain: ref.current?.name,
-      })}
       deleteResource={() =>
         domain?.delete().then(async () => {
           await domains?.revalidate?.();
-          close();
+          onSuccess();
         })
       }
-      breadcrumbTitle={localizationKeys('organizationProfile.profilePage.domainSection.title')}
```
Comment (anagstef) [inline packages/clerk-js/src/ui/components/OrganizationProfile/RemoveDomainForm.tsx:78]: Should we also remove this localizaiton key?

Comment (desiprisg) [reply packages/clerk-js/src/ui/components/OrganizationProfile/RemoveDomainForm.tsx:78]: I'll restructure all the keys in another PR soon.

**Pull request merged**